src/styles/theme/defaultTheme.ts

```diff
@@ -82,6 +82,15 @@ declare module '@mui/material/styles' {
 }
 
 export const defaultTheme = createTheme({
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 600,
+      md: 1000, // default is 900
```

Comment:
Do we have to override all of them, in order to override this single one? Can't we do something like 
```
values: {
   ...defaultValues,
   md: 900,
}
```

?